packages/back-end/src/util/sql.ts

```diff
@@ -12,6 +12,10 @@ Object.keys(helpers).forEach((helperName) => {
   Handlebars.registerHelper(helperName, helpers[helperName]);
 });
 
+// Re-export FormatDialect and format for backward compatibility
+export type { FormatDialect };
+export { format };
+
```

Comment:
This is a bit of a weird pattern - we were previously exporting these from here. This makes the diff much smaller, but happy to change if we don't want to introduce this pattern.